### DISCUSSION TITLE
Attachment column images are not display

### DISCUSSION THREAD

**Discussion by Arronaxx** [Budibase/budibase#6064] [Help] [answered]
Hello, excuse me for my noob question, i am low-code platform user ಠ_ಠ

I google a lot, i read forums, i watched budibase youtube channel, but i can't figure out the simpliest thing:

1. I create attachment column
![111](https://user-images.githubusercontent.com/103760873/170373519-20b75c63-2b5d-4976-a019-80b3301c4a17.png)

2. Then i upload image
![222](https://user-images.githubusercontent.com/103760873/170373750-98475c60-b12e-4ef0-a2fe-ecea31560e7a.png)

3. Then i try to display current element name - that's work correct
![333](https://user-images.githubusercontent.com/103760873/170373773-41c9d116-d770-40f9-987e-74ee8cfba3eb.png)

4. Then i try to display image attachment
![444](https://user-images.githubusercontent.com/103760873/170373851-880c3a2e-0309-47e6-8bcf-ca3a7727b8c9.png)

What am i doing wrong? Am i make a logical mistake somewhere (i try to learn from templates library)? What's the ways to display images which attach in attachments column? What's the ways to upload and display images in general?

**Comment by melohagan** [comment, marked as answer]
Hi @Arronaxx 

Thank you for using Budibase.

I've had a look and in this case there is a little more complexity. 
As an attachment can be any file type and any number of files, Budibase cannot assume that it's a single image within the image component. 
The image component takes a URL, so we need to access that from the attachment object.

I have added a page to the documentation that explains the process: https://docs.budibase.com/docs/image#displaying-images-from-an-attachment

Please let me know if you need further clarification!

**Reply by Arronaxx** [reply]
Yeah, now it works! I am very grateful to you. Have a good day!

**Reply by melohagan** [reply]
You're welcome, have a good day yourself :)
